package.json

```diff
@@ -176,9 +176,10 @@
     "react-input-autosize": "1.1.0",
     "react-input-range": "1.2.1",
     "react-markdown": "2.4.2",
-    "react-redux": "4.4.5",
+    "react-redux": "^5.0.5",
     "react-router": "2.0.0",
     "react-router-redux": "4.0.4",
+    "react-router-dom": "^4.2.2",
```

Comment:
I think we want to leave out the ^ part to match the other packages.  I think the @elastic/kibana-operations team has a flow for updating these packages.